noria-server/dataflow/src/node/mod.rs

```diff
@@ -110,9 +110,9 @@ impl Node {
         self.sharded_by = s;
     }
 
-    /// Add a reader replica to this node.
-    pub fn add_replica(&mut self, ni: NodeIndex) {
-        self.replicas.push(ni)
+    /// The node being added is a reader for this node.
```

Comment:
What is "the node being added"? Do you mean `ni`?

Reply:
Yes, modified the comment.